<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1400</width>
      <height>900</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="exp01">
     <attribute name="title">
      <string>Exp01</string>
     </attribute>
     <widget class="QLabel" name="exp01Show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>768</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>1090</x>
        <y>30</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="exp02">
     <attribute name="title">
      <string>Exp02</string>
     </attribute>
     <widget class="QLabel" name="exp02Show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>768</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <zorder>exp01Show</zorder>
     <zorder>exp02Show</zorder>
    </widget>
    <widget class="QWidget" name="exp03">
     <attribute name="title">
      <string>Exp03</string>
     </attribute>
     <widget class="QLabel" name="exp03Show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>768</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="exp04">
     <attribute name="title">
      <string>Exp04</string>
     </attribute>
     <widget class="QLabel" name="exp04Show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>768</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="exp05">
     <attribute name="title">
      <string>Exp05</string>
     </attribute>
     <widget class="QLabel" name="exp05Show">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1280</width>
        <height>768</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionopen_image"/>
    <addaction name="actionclose"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionopen_image">
   <property name="text">
    <string>open image</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
